pkg/serviceaccounts/controllers/deleted_token_secrets.go

```diff
@@ -1,101 +0,0 @@
-package controllers
-
-import (
-	"context"
-	"fmt"
-	"time"
-
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/klog/v2"
-
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	informers "k8s.io/client-go/informers/core/v1"
-	kclientset "k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/cache"
-)
-
-// DockercfgTokenDeletedControllerOptions contains options for the DockercfgTokenDeletedController
-type DockercfgTokenDeletedControllerOptions struct {
-	// Resync is the time.Duration at which to fully re-list secrets.
-	// If zero, re-list will be delayed as long as possible
-	Resync time.Duration
-}
-
-// NewDockercfgTokenDeletedController returns a new *DockercfgTokenDeletedController.
-func NewDockercfgTokenDeletedController(secrets informers.SecretInformer, cl kclientset.Interface, options DockercfgTokenDeletedControllerOptions) *DockercfgTokenDeletedController {
```

Comment:
Without this controller, what deletes the token secrets when the dockercfg secret is deleted later on?